chat/rich-links/src/RichLink/RichLinkBadge/RichLinkBadge.styles.ts

```diff
@@ -0,0 +1,139 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { fontFamilies, fontWeights } from '@leafygreen-ui/tokens';
+
+import { Variant } from './RichLinkBadge.types';
+
+export const baseStyles = css`
+  font-family: ${fontFamilies.default};
```

Comment:
You might not need to specify the font family and font weight, those should already be defined in `Card`.